recipes/meos/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+sed -i "s,/usr/local,${PREFIX},g" ./meos/CMakeLists.txt
+
+mkdir -p build && cd build
+
+cmake ${CMAKE_ARGS} \
+      -D CMAKE_BUILD_TYPE=Release \
+      -D CMAKE_INSTALL_PREFIX=${PREFIX} \
+      -D CMAKE_INSTALL_LIBDIR=lib \
+      -D MEOS=on \
+      -D GSL_INCLUDE_DIR=${PREFIX}/include \
+      -D GSL_LIBRARY=${PREFIX}/lib/libgsl.so \
+      -D GSL_CBLAS_LIBRARY=${PREFIX}/lib/libgslcblas.so \
+      -D PROJ_INCLUDE_DIRS=${PREFIX}/include \
+      -D PROJ_LIBRARIES=${PREFIX}/lib/libproj.so \
+      -D GEOS_INCLUDE_DIR=${PREFIX}/include \
+      -D GEOS_LIBRARY=${PREFIX}/lib/libgeos_c.so \
+      -D JSON-C_INCLUDE_DIRS=${PREFIX}/include/json-c \
+      -D JSON-C_LIBRARIES=${PREFIX}/lib/libjson-c.so \
+      ${SRC_DIR}
+
+make -j
+make install
```

Comment:
Is there a `make check` or `ctest` that we can run here?

Reply:
I don't think there is now, but I need @estebanzimanyi to confirm or deny this.

Reply:
`ctest` is used for all the CI tests in the MobilityDB (PostgreSQL) binding, e.g.,
https://github.com/estebanzimanyi/MobilityDB/actions/runs/7252347113/job/19756608599
We didn't used `ctest` for the PyMEOS binding.

Reply:
Indeed, we don't have a check for MEOS, since the library is tested through MobilityDB.
Will this be an issue? If so, we could activate the testing in the `pymeos` recipe, which would report any issue with the `meos` library.